package.json

```
@@ -29,9 +29,20 @@
     "node-mocks-http": "^1.8.0"
   },
   "devDependencies": {
+    "husky": ">=4",
+    "lint-staged": ">=10",
```

Let's use caret range for these, just in case a future major version of `husky` or `lint-staged` breaks the API:

```suggestion
    "husky": "^4.2.5",
    "lint-staged": "^10.2.11",
```